test/integration/integration_test.go

```diff
@@ -68,8 +69,30 @@ func TestMain(m *testing.M) {
 	runner := util.MinishiftRunner{CommandPath: minishiftBinary}
 	if runner.IsCDK() {
 		godogTags += "~minishift-only"
+		isoName = "rhel"
+		fmt.Println("Test run using CDK binary with RHEL iso.")
 	} else {
 		godogTags += "~cdk-only"
+		switch os.Getenv("MINISHIFT_ISO_URL") {
+		case "b2d":
+			fmt.Println("Test run using Boot2Docker iso image.")
+			isoName = "b2d"
+		case "centos":
+			fmt.Println("Test run using CentOS iso image.")
+			isoName = "centos"
+		case "":
+			fmt.Println("Test run using Boot2Docker iso image.")
+			isoName = "b2d"
+		default:
+			envIso := os.Getenv("INTEGRATION_ISO")
```

Comment:
I guess here is where things fall apart using the MINISHIFT_ISO_URL. Is this because you are using file://... when running integration tests. Are you requiring in this case to set another variable? I guess here the dynamic approach would be better. Would it not also be possible to do a regexp check of the iso name and decide based on that. Seems "nicer" than introducing a new variable.